config/recipes/lustre-client/manifests/1000-driver-container.yaml

```diff
@@ -58,10 +63,10 @@ data:
 
     mkdir -p /mnt/fsx
 
-    trap "echo 'Caught signal'; exit 1" HUP INT QUIT PIPE TERM
-    trap "unmount" EXIT
+    sleep infinity & PID=$!
+    trap "kill $PID; unmount" HUP INT QUIT PIPE TERM EXIT
+    wait
```

Comment:
I think we can leverage here systemd like in the simple-kmod example. 
```
ExecStartPre=/usr/bin/bash -c "kmods-via-containers build %i %v"
ExecStart=/usr/bin/bash -c "kmods-via-containers load %i %v"
ExecReload=/usr/bin/bash -c "kmods-via-containers reload %i %v"
ExecStop=/usr/bin/bash -c "kmods-via-containers unload %i %v"
```

We wouldn't need a wait or trap kill or sleep infinity

Reply:
@zvonkok I added a new commit with lustre-client switched to using kvc. See also https://github.com/openshift-psap/kvc-lustre-client. Please look it over when you get a chance to make sure it looks okay. It seems to work for me to load the kmod. 

However, I don't think it unloads the kernel module when the driver container is deleted or after `oc delete specialresource/lustre-client`. Not sure if there is more I need to do to the recipe to make this work, or if this is a problem we need to address in kvc itself.